viadot/sources/cloud_for_customers.py

```diff
@@ -14,7 +14,7 @@
 from requests.packages.urllib3.util.retry import Retry
 
 
-def map_columns(url=None):
+def map_columns(url: str = None) -> Dict:
```

Comment:
If you don't know the type of the elements, use can use Dict[str, Any]. However, here it's probably `Dict[str, str]`?